src/hardware/gus.cpp

```diff
@@ -194,6 +194,25 @@ class Voice {
 static void GUS_TimerEvent(uint32_t t);
 static void GUS_DMA_Event(uint32_t val);
 
+// Reset Register (4Ch)
```

Comment:
Minor typo in the commit message: "This replaces three standard alone variable"

Standalone?

Reply:
oops! fixing.